Sampled/UI/Scenes/AppScene.swift

```diff
@@ -15,7 +15,7 @@ struct AppScene: Scene {
     LibraryScene()
       .environment(library)
 
-    LibraryInfoScene()
+    HistoryScene()
```

Comment:
Is LibraryInfoScene gone? I think the UtilityWindow approach looked nice. Can you add it back?

I don't think HistoryScene should be a scene, given that the app may support multiple library windows at once. The "Up Next" inspector displays a list of tracks in the queue. I think you could amend it so an "Upcoming" and "Previous" tab exists with this information, instead. It may require you to redesign the UI you've created, however.